drivers/usb/tests/TESTS/host_tests/usb_device_hid.py

```
@@ -42,13 +43,6 @@
     # Use a default backend on other platforms.
     USB_BACKEND = None
 
-try:
-    import hid
-except ImportError:
```

This check is actually kinda counterproductive, because it causes some test cases to be silently skipped if the hid package isn't installed.  I'd much rather bail out on the test so that the user knows they are missing a package.